src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenServiceCollectionExtensions.cs

```diff
@@ -44,47 +44,22 @@ public static IServiceCollection AddSwaggerGen(
         return services;
     }
 
-    public static void ConfigureSwaggerGen(
-        this IServiceCollection services,
-        Action<SwaggerGenOptions> setupAction)
+#if NET
+    public static IServiceCollection AddSwaggerGenMinimalApisJsonOptions(this IServiceCollection services)
```

Comment:
Could you add something to the README showing how to use these two new methods please?